### ISSUE TITLE
how to use it

### ISSUE THREAD

**Issue by luosu666** [yeyeye613/hdu_memorize_words#1]
is it on computer？

**Comment by yeyeye613** [comment]
yes,I will update readme.md within this week and try to make it easier to use（I'm also a newbie）

**Comment by luosu666** [comment]
it need install chrome drivers.I It failed before because I didn't install it.Your script works very well.
